Automattic/jetpack
# Social: Move the admin upsell

Fixes https://github.com/Automattic/jetpack-reach/issues/382

## Proposed changes:

* Moved the nudge to the new place
* Changed phrasing so it makes sense on the Basic plan too
* Improved tooltip message

### Other information:


## Jetpack product discussion


p1717664546747509-slack-C02JJ910CNL
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Go to the admin page
* With no plans, you should see the nudge
* With the basic plan you should see the nudge
* With the Advanced plan or the new V1 plan you should not see the nudge

![CleanShot 2024-06-06 at 11 52 04 png](https://github.com/Automattic/jetpack/assets/36671565/df01f767-fea1-4498-9459-c345b191fd88)
